arangod/Aql/ExecutionNode/CollectNode.h

```diff
@@ -26,6 +26,7 @@
 #include "Aql/CollectOptions.h"
 #include "Aql/ExecutionNode/ExecutionNode.h"
 #include "Aql/ExecutionNodeId.h"
+#include "Basics/ResourceUsage.h"
```

Comment:
Please forward declare the ResouceMonitor here and only include in the c++ file.
Saves some recompiles later on.